pkg/cmd/cockroach-short/main.go

```diff
@@ -20,8 +20,10 @@ package main
 import (
 	_ "github.com/cockroachdb/cockroach/pkg/ccl" // ccl init hooks
 	"github.com/cockroachdb/cockroach/pkg/cli"
+	"github.com/cockroachdb/cockroach/pkg/testutils/bazelcodecover"
 )
 
 func main() {
+	bazelcodecover.MaybeInitCodeCoverage()
 	cli.Main()
```

Comment:
1. Why not just put `MaybeInitCodeCoverage()` in `cli.Main()`?
2. Why not put it in `cockroach-oss` as well?